### ISSUE TITLE
can't change root password when DC installed on disk

### ISSUE THREAD

**Issue by YanChii** [erigones/esdc-ce#516]
```
# ll /etc/{passwd,shadow}
-rw-r--r--   1 root     sys         1065 Mar 18  2018 /etc/passwd
-r--------   1 root     root         560 Jun  9 19:15 /etc/shadow

# passwd root
New Password: 
Re-enter new Password: 
passwd: Unexpected failure. Password database unchanged.
Permission denied

# set-root-password 
New Password: 
Re-enter new Password: 
mv: cannot rename /etc/shadow.new to /etc/shadow: Device busy
```

**Comment by YanChii** [comment]
Workaround for current platform:
```
# umount /etc/shadow
# set-root-password
```

**Comment by YanChii** [comment]
Fixed in `v4.4`
